lib/Service/DropboxStorageAPIService.php

```diff
@@ -247,48 +238,49 @@ private function getFile(string $accessToken, string $refreshToken, string $clie
 			$saveFolder = $topFolder;
 		} else {
 			$saveFolder = $this->createAndGetFolder($dirName, $topFolder);
-		}
-		try {
-			$fileExists = $saveFolder->nodeExists($fileName);
-		} catch (ForbiddenException $e) {
-			return null;
-		}
-		if (!is_null($saveFolder) && !$fileExists) {
-			try {
-				$savedFile = $saveFolder->newFile($fileName);
-				$resource = $savedFile->fopen('w');
-			} catch (NotFoundException $e) {
+			if (is_null($saveFolder)) {
 				$this->logger->warning(
-					'Dropbox error, can\'t create file "' . $fileName . '" in "' . $saveFolder->getPath() . '"',
+					'Dropbox error, can\'t create directory "' . $dirName . '"',
 					['app' => $this->appName]
 				);
 				return null;
 			}
-			$res = $this->dropboxApiService->downloadFile(
-				$accessToken, $refreshToken, $clientID, $clientSecret, $userId, $resource, $fileItem['id']
+		}
+		if ($saveFolder->nodeExists($fileName)) {
+			return null;
+		}
```

Comment:
```suggestion
		try {
			if ($saveFolder->nodeExists($fileName)) {
				return null;
			}
		} catch (ForbiddenException $e) {
			return null;
		}
```
In some cases, this is needed. For example when importing files named `.htaccess`.